src/js/containers/award/AwardContainer.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+  * AwardContainer.jsx
+  * Created by Emily Gullo 01/19/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Award from 'components/award/Award';
+
+import * as SearchHelper from 'helpers/searchHelper';
+import * as awardActions from 'redux/actions/award/awardActions';
+
+const propTypes = {
+    setSelectedAward: React.PropTypes.func,
+    params: React.PropTypes.object
+};
+
+class AwardContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.getSelectedAward = this.getSelectedAward.bind(this);
+    }
+
+    getSelectedAward() {
+        const input = this.props.params.awardId;
+
+        if (this.selectedAwardRequest) {
+            // A request is currently in-flight, cancel it
+            this.selectedAwardRequest.cancel();
+        }
+
+        const selectedAwardParams = {
+            filters: [
+                {
+                    field: "piid",
+                    value: input,
+                    operation: "equals"
+                }
+            ]
+        };
+
+        this.selectedAwardRequest = SearchHelper.fetchAward(selectedAwardParams);
+
+        this.selectedAwardRequest.promise
+            .then((res) => {
+                const data = res.data;
+                const awardData = data.results[0];
```

Comment:
The container should also be able to handle a condition where there is no award with the given ID.